stake-pool/program/src/instruction.rs

```diff
@@ -30,6 +30,7 @@ pub enum StakePoolInstruction {
     ///   7. `[]` Clock sysvar
     ///   8. `[]` Rent sysvar
     ///   9. `[]` Token program id
+    ///  10. `[s]` (Optional) Deposit authority that must sign all deposits
```

Comment:
The deposit authority shouldn't need to sign `Initialize`.  This'll will save a transaction signature fee for the user

Reply:
Can you also document here the behaviour when the deposit authority is not provided

Reply:
Will do, I wanted to keep things safe to be sure that the depositor was around on creation.